### DISCUSSION TITLE
Unable to use ipV6 in DNS list

### DISCUSSION THREAD

**Discussion by aliekiddo** [AdguardTeam/AdGuardHome#4822] [Q&A]
I'm getting an error when adding an ipv6 or assigning a port to my DNS settings. 
<img width="741" alt="Screenshot 2022-08-07 at 19 43 20" src="https://user-images.githubusercontent.com/7974754/183306444-eee99e94-77cc-487d-8667-310bff3ac58e.png">

What am I doing wrong?

**Comment by ppfeufer** [comment]
I guess it's the double colon (`::`)

AGH might not like it. Had a similar issue when I tried to get AGH to listen on an IPv6 with a double colon.

**Comment by aliekiddo** [comment]
It doesn't seem to like colons full stop.
I'm getting the same error from Google's IPv6
2001:4860:4860:0:0:0:0:8888

**Reply by juanfrancisco-18** [reply]
tampoco me funciona

**Reply by telometto** [reply]
@juanfrancisco-18 Habla ingles, por favor.

**Comment by ppfeufer** [comment]
Erm, I'm just seeing it, you gotta write IPv6 addresses in `[]`

Google for example:

```
# Google IPv6
tls://[2001:4860:4860::8888]
tls://[2001:4860:4860::8844]
https://[2001:4860:4860::8888]/dns-query
https://[2001:4860:4860::8844]/dns-query
```

**Reply by juanfrancisco-18** [reply]
no funciona asi tampoco

**Comment by aliekiddo** [comment]
I'm getting the same error for that too. I think the square brackets just denote when you're querying a specific address
"[/example.local/]94.140.14.140:an upstream for specific domains;"

I think this may be more to do with my setup not being able to communicate over IPv6 for some reason rather than syntax as I swear I've been able to use IPv6 in the upstreams in a previous setup.

**Comment by juanfrancisco-18** [comment]
hola   como se puede colocar el dns en ipv6?
